packages/data/src/components/with-select/test/index.js

```diff
@@ -18,7 +18,19 @@ import withDispatch from '../../with-dispatch';
 import { createRegistry } from '../../../registry';
 import { RegistryProvider } from '../../registry-provider';
 
+/* eslint-disable @wordpress/wp-global-usage */
 describe( 'withSelect', () => {
+	const initialScriptDebug = globalThis.SCRIPT_DEBUG;
+
+	beforeAll( () => {
+		// Do not run HOC in development mode; it will call `mapSelect` an extra time.
+		globalThis.SCRIPT_DEBUG = false;
+	} );
+
+	afterAll( () => {
+		globalThis.SCRIPT_DEBUG = initialScriptDebug;
+	} );
```

Comment:
If we need this more and more in the future, we might consider abstracting it into an opt-in global jest setup/teardown configuration. However, for just 3 instances, it will be an unnecessary abstraction.